DQMOffline/JetMET/python/jetAnalyzer_cfi.py

```diff
@@ -122,79 +122,79 @@
 
 
 jetDQMAnalyzerAk4PFCleaned=jetDQMAnalyzerAk4PFUncleaned.clone(
-    JetCleaningFlag = cms.untracked.bool(True),
-    filljetHighLevel = cms.bool(False),
+    JetCleaningFlag = True,
+    filljetHighLevel = False,
     jetAnalysis=jetDQMParameters.clone(
-        ptThreshold = cms.double(20.),
-        asymmetryThirdJetCut = cms.double(30),
-        balanceThirdJetCut = cms.double(0.2),
+        ptThreshold = 20.,
+        asymmetryThirdJetCut = 30,
+        balanceThirdJetCut = 0.2,
         ),
-    METCollectionLabel     = cms.InputTag("pfMet"),
+    METCollectionLabel     = "pfMet"
 )
 
 jetDQMAnalyzerAk4PFCHSCleaned=jetDQMAnalyzerAk4PFCleaned.clone(
-    filljetHighLevel =cms.bool(True),
-    JetCorrections = cms.InputTag("dqmAk4PFCHSL1FastL2L3ResidualCorrector"),
-    jetsrc = cms.InputTag("ak4PFJetsCHS"),
-    METCollectionLabel     = cms.InputTag("pfMETT1"),
-    InputMVAPUIDDiscriminant = cms.InputTag("pileupJetIdEvaluatorCHSDQM","fullDiscriminant"),
-    InputCutPUIDDiscriminant = cms.InputTag("pileupJetIdEvaluatorCHSDQM","cutbasedDiscriminant"),
-    InputMVAPUIDValue = cms.InputTag("pileupJetIdEvaluatorCHSDQM","fullId"),
-    InputCutPUIDValue = cms.InputTag("pileupJetIdEvaluatorCHSDQM","cutbasedId"),
-    fillCHShistos =cms.bool(True),
+    filljetHighLevel = True,
+    JetCorrections = "dqmAk4PFCHSL1FastL2L3ResidualCorrector",
+    jetsrc = "ak4PFJetsCHS",
+    METCollectionLabel     = "pfMETT1",
+    InputMVAPUIDDiscriminant = "pileupJetIdEvaluatorCHSDQM : fullDiscriminant",
```

Comment:
I am not sure about the spaces before and after the : 
You'd better remove them